Gem/Code/Source/Components/NetworkAnimationComponent.cpp

```diff
@@ -100,16 +100,16 @@ namespace MultiplayerSample
 
         if (m_velocityParamId == InvalidParamIndex)
         {
-            m_velocityParamId = m_animationGraph->FindParameterIndex(GetVelocityParamName().c_str());
-            m_aimTargetParamId = m_animationGraph->FindParameterIndex(GetAimTargetParamName().c_str());
-            m_crouchParamId = m_animationGraph->FindParameterIndex(GetCrouchParamName().c_str());
-            m_aimingParamId = m_animationGraph->FindParameterIndex(GetAimingParamName().c_str());
-            m_shootParamId = m_animationGraph->FindParameterIndex(GetShootParamName().c_str());
-            m_jumpParamId = m_animationGraph->FindParameterIndex(GetJumpParamName().c_str());
-            m_fallParamId = m_animationGraph->FindParameterIndex(GetFallParamName().c_str());
-            m_landParamId = m_animationGraph->FindParameterIndex(GetLandParamName().c_str());
-            m_hitParamId = m_animationGraph->FindParameterIndex(GetHitParamName().c_str());
-            m_deathParamId = m_animationGraph->FindParameterIndex(GetDeathParamName().c_str());
+            m_velocityParamId = static_cast<uint32_t>(m_animationGraph->FindParameterIndex(GetVelocityParamName().c_str()));
```

Comment:
Do we prefer `az_numeric_cast`?

Reply:
in this case I can switch it, but was leaving it to the code owners to do what is best... e.g. a better solution could be to change those Ids to the proper type

Reply:
you can leave this as is, though my preference would be to update these types in the class definition to match the new emfx return value type (AZStd::size_t?)

Reply:
actually yeah, since the invalid return type is now (uint64_t)(-1).. we should be using the EMFX types here to avoid bugs.

Reply:
changed